package.json

```diff
@@ -21,6 +21,7 @@
     "start:dev": "webpack-dev-server --config ./webpack.dev.ts --progress false",
     "start:dev-cloud": "cross-env CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:4000 webpack-dev-server --config ./webpack.dev.ts",
     "start:docker": "yarn generate && yarn build:vendor && yarn run start:dev",
+    "start:kind": "yarn build:vendor && NODE_ENV=development UI_SHA=local_dev_mode BASE_PATH=/ API_BASE_PATH=/ PORT=8080 PUBLIC=https://twodotoh.a.influxcloud.dev.local/ CLOUD_URL=/auth yarn start",
```

Comment:
how do i change twodotoh.a.influxcloud.dev.local to "cool.kids.table"?

Reply:
hmm it's hardcoded in several places in k8s-idpe so I don't think it's expected that you would change this

Reply:
perhaps an improvement for down the road once all the cool kids are using k8s-idpe ;)